### DISCUSSION TITLE
Loading Snippets from File

### DISCUSSION THREAD

**Discussion by liris8** [artisticat1/obsidian-latex-suite#148] [Q&A] [answered]
First: THANK U. This plugin is something I've been waiting for so long. 

Second, I don't have a very powerful computer background and I feel sometimes out of the place when doing some technical stuff... I just wanted to ask: 

**How can I add a file of snippets?**, meaning:  **which extension the file must be?** Have the snippets to be written in the **same format** as the ones that come in the plugging settings?

The thing is that I do most of my workflow in VS Code where I have a well snippet integration thanks to of course _Gilles Castel_ and more percisely to _sleepymac_ on his brilliant post https://github.com/sleepymalc/VSCode-LaTeX-Inkscape. There he exposes most of the snippets that I'm using and the format in which I write them. If this format and document extension (.hsnips) would not work, do you have some advice on how to fix it?

Thank u very much in advance again, you are amazing!

**Comment by superle3** [comment, marked as answer]
I dont this was available when this was asked? But it is now and the file is interpreted as javascript.

We import the default export so 
```js

export default [...]
```

Is valid.
See https://github.com/superle3/obsidian-latex-suite/blob/feat/correct-docs/DOCS.md#snippet-files for more info. As to the extension, it can be anything.
